app/models/opendap_configuration.rb

```diff
@@ -160,22 +160,30 @@ def info_urls_for(granule)
     end
   end
 
-  def download_urls_for(granule)
+  def download_urls_for(granule, is_cwic=false)
     if @can_subset
       [@template.expand(decorate(granule).merge(@template_params))]
     else
-      links_for(granule, 'enclosure', /\/data/)
+      if is_cwic
+        Array.wrap(links_for(granule['link'], 'enclosure', /enclosure/).first)
+      else
+        links_for(granule['links'], 'enclosure', /\/data/)
+      end
     end
   end
 
-  def browse_urls_for(granule)
-    links_for(granule, /\/browse/)
+  def browse_urls_for(granule, is_cwic=false)
```

Comment:
This needs to be updated to match the logic in the browse URL branch.

Also, for whatever reason, I'm not seeing browse links show up on the download page for my test collection (http://edsc.dev/search?cmr_env=uat&q=USGS_EDC_EO1_ALI&ff=Int%27l+%2F+Interagency)